File: algorithms/linear-search.js

function linearSearch(array, number) {
    for (let i = 0; i < array.length; i++){
        if (array[i] === number){
            return i;
        }
    }

    return -1;
}

const array = [10, 4, 5, 1, 9, 7, 6];

console.log(linearSearch(array, 5));

File: python/algorithms/linear_search.py
def linear_search(lists, x):
    for i in range(len(lists)):
        if lists[i] == x:
            return i
    return -1


lists = [1, 4, 3, 6, 8, 7, 2, 5]
print(linear_search(lists, 3))
